index.html

```diff
@@ -132,6 +132,7 @@ <h2 id="overview">What is EditorConfig?</h2>
     <li><a href="https://github.com/JetBrains/intellij-community/tree/master/plugins/editorconfig"><img src="logos/rubyMine.png" alt="RubyMine logo"><span>RubyMine</span></a></li>
     <li><a href="https://www.sourcelair.com/features/editorconfig"><img src="logos/sourcelair.png" alt="SourcLair logo"><span>SourcLair</span></a></li>
     <li><a href="https://tortoisegit.org/"><img src="logos/TortoiseGit.png" alt="TortoiseGit logo"><span>TortoiseGit</span></a></li>
+    <li><a href="https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options"><img src="logos/visualstudio-pro.png" alt="Visual Studio Pro logo"><span>Visual Studio Professional</span></a></li>
```

Comment:
It should probably be noted somewhere that this is only version 2017 and above (future).